### ISSUE TITLE
Cannot enable SAS in v1.7.0

### ISSUE THREAD

**Issue by murphm8** [RemoteTechnologiesGroup/RemoteTech#639]
1. Clean KSP install and installed RemoteTech via CKAN. 
2. Created a rocket with the Probodobodyne OKTO and an antenna. It shows connected and I can see the flight computer.
3. I am unable to enable SAS. When I click on SAS on the nav ball the message "No Target" pops up on the screen. I attempted to right click the probe and "control from here" but that didn't change anything.

I could totally be missing something in how to use remote tech. I searched around and didn't see a similar issue anywhere so I figured I'd post here.

**Comment by murphm8** [comment]
So using the shortcut for SAS "T" works, but clicking doesn't. Is that expected?

**Comment by Zhetaan** [comment]
Yes.  You can use the mouse to set or unset a target, but the navball is informational only; nothing on it is clickable (note that the SAS modes _next to_ the navball _are_ clickable).  Since the SAS button on the navball is not a possible target, clicking on it will only unset any target you have--or, if you don't have a target set, simply confirms that with the message, 'No Target'.  This is all stock behaviour.

Where RemoteTech gets involved is when you use the flight computer to set your ship's orientation.  To rotate your ship, the computer will deactivate SAS, and you won't be able to activate it until you turn the flight computer off (click the 'X' in the expanded FC panel; 'KILL' is equivalent to 'Stability Assist' and is not the same as 'Off').  This is also expected behaviour:  if you can't activate SAS with 'T', then check your flight computer.

**Comment by hvacengi** [comment]
>the navball is informational only; nothing on it is clickable (note that the SAS modes next to the navball are clickable).

Actually, in the current version both the SAS and RCS indicators are clickable buttons for enabling their respective control.  I just verified that in stock KSP a single click on the SAS icon will activate, or deactivate, the ship's SAS.

**Comment by Zhetaan** [comment]
Well, that's interesting.  I wonder when they made those buttons clickable; it definitely was not always that way, but the changelogs I just looked through don't mention it.  Nevertheless, I stand corrected.

**Comment by hvacengi** [comment]
I suspect it has to do with the UI update for KSP 1.1.x.  I know that at one point the devnotes said that they had to re-write the navball UI code.  It always seemed odd to me in the past that they were not buttons, given that you had lights/gear/brakes buttons in the top UI.

**Comment by Zhetaan** [comment]
That makes sense.  Once they added the SAS mode toggles, I'm sure the developers were looking for other ways to give the navball a more overall-interactive interface.

**Comment by neitsa** [comment]
This is actually an intended behavior (from what I see in the code), the buttons are locked **unless** you have local control.

The interesting part is [located here](https://github.com/RemoteTechnologiesGroup/RemoteTech/blob/d422bb87c330f86bcad1ae6f7fb5ab21724c9e2a/src/RemoteTech/RTCore.cs#L99). The buttons are locked except if the vessel can be locally controlled.
As you can see [here](https://github.com/RemoteTechnologiesGroup/RemoteTech/blob/d422bb87c330f86bcad1ae6f7fb5ab21724c9e2a/src/RemoteTech/RTCore.cs#L175) the `GetLocks()` function also locks the SAS button (which means you have to use the SAS key to enable SAS).

If you think this is still an issue, please make a comment, otherwise I'll close this issue in a month.

**Comment by neitsa** [comment]
Revising my previous comment and chaging from "Won't fix" to "Verified bug": the problem with locked buttons seems to have appeared circa KSP 1.1.x. I haven't been able to understand what the locks are for when you have a working connection, but they are still here and nobody seems to have has an idea why.

I'll try to fix this issue as it is really bothering me (the problem being that a fix (removing button locks) might introduce some bugs).

**Comment by dmayan-ss** [comment]
As 1.9.7 on KSP 1.10.1 this is not solved?

**Comment by TeusDev** [comment]
this bug is still part of RT code ? cannot use action groups or UI at all in unmanned.

**Comment by nymda** [comment]
Is this going to be fixed? Still an issue 10 years on
